rst/troubleshooting.rst

```diff
@@ -310,8 +310,8 @@ To change instance advertise URI you have to perform these actions:
 
     ..  code-block:: bash
 
-        echo "return box.info().uuid" | tarantoolctl connect \
-        unix/:/var/run/tarantool/<app-name>.<instance-name>.control
+        echo "return box.info().uuid" | tt connect \
+        unix/:/var/run/tarantool/<app-name>.<instance-name>.control -f
```

Comment:
This won't work because `unix/:` is incorrect URI format (https://datatracker.ietf.org/doc/html/rfc3986). it must be:
`unix://var/run/tarantool/<app-name>.<instance-name>.control`
or simply:
`./var/run/tarantool/<app-name>.<instance-name>.control`

Reply:
Thanks, tested ad fixed in all ``tt connect`` calls on this page. Only this worked for me on CentOS 7:
`/var/run/tarantool/<app-name>.<instance-name>.control` (without a starting dot)

Reply:
Oh, these are absolute paths... Yes, dot is not required for absolute paths. Simple format:
`/var/run/tarantool/example.control`
With schema:
`unix:///var/run/tarantool/example.control`